<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpacerDialog_UI</class>
 <widget class="QDialog" name="SpacerDialog_UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>111</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add space</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="KRestrictedLine" name="space_duration">
       <property name="inputMask">
        <string>99:99:99:99; </string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="position_up">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>18</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
         <property name="arrowType">
          <enum>Qt::UpArrow</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="position_down">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>18</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
         <property name="arrowType">
          <enum>Qt::DownArrow</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="clip_filesize_3">
     <property name="text">
      <string>Track</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>218</width>
       <height>2</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="clip_filesize_2">
     <property name="text">
      <string>Duration</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="KComboBox" name="track_number"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KRestrictedLine</class>
   <extends>KLineEdit</extends>
   <header>krestrictedline.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SpacerDialog_UI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SpacerDialog_UI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
